packages/phone-number-privacy/signer/azure-templates/container-parameters.json

```diff
@@ -0,0 +1,92 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "containerName": {
+            "value": "TODO"
+        },
+        "location": {
+            "value": "TODO"
+        },
+        "imageType": {
+            "value": "Public"
+        },
+        "imageName": {
+            "value": "us.gcr.io/celo-testnet/celo-monorepo:phone-number-privacy-rc1"
+        },
+        "osType": {
+            "value": "Linux"
+        },
+        "numberCpuCores": {
+            "value": "2"
+        },
+        "memory": {
+            "value": "2"
+        },
+        "restartPolicy": {
+            "value": "Always"
+        },
+        "environmentVariables": {
+            "value": [
+                {
+                    "name": "BLOCKCHAIN_PROVIDER",
+                    "value": "https://rc1-forno.celo-testnet.org"
+                },
+                {
+                  "name": "SERVER_PORT",
+                  "value": "80"
+                },
+                {
+                    "name": "DB_TYPE",
+                    "value": "postgres"
+                },
+                {
+                    "name": "DB_HOST",
+                    "value": "TODO"
+                },
+                {
+                    "name": "DB_USERNAME",
+                    "value": "TODO"
+                },
+                {
+                    "name": "DB_PASSWORD",
+                    "secureValue": "TODO"
+                },
+                {
+                    "name": "DB_DATABASE",
+                    "value": "phoneNumberPrivacy"
+                },
+                {
+                    "name": "KEYSTORE_TYPE",
+                    "value": "AzureKeyVault"
+                },
+                {
+                    "name": "KEYSTORE_AZURE_VAULT_NAME",
+                    "value": "TODO"
+                },
+                {
+                    "name": "KEYSTORE_AZURE_SECRET_NAME",
+                    "value": "TODO"
```

Comment:
Optionally for these TODO params, you can remove them from here and force the user to specify them at the command line. It might make it clearer which params need to be overridden 
https://docs.microsoft.com/en-us/cli/azure/deployment/group?view=azure-cli-latest#az-deployment-group-create-examples (see last example)
```
az deployment group create \
  --resource-group $RESOURCE_GROUP \
  --template-file container-template.json \
  --parameters @container-parameters.json
  --parameters 
     KEYSTORE_AZURE_VAULT_NAME=<YOUR_VAULT_NAME>
     KEYSTORE_AZURE_SECRET_NAME=<YOUR_SECRET_NAME>
     ...
```